src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -350,7 +356,7 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
             string graphType(type);
             graphType = utils.trim_copy(graphType, " \f\n\r\t\v");
 
-            std::unordered_set<std::string> s = {"1", "2", "3"};
+            std::unordered_set <std::string> s = {"1", "2", "3"};
```

Comment:
Rather than having single letter variable names can we have more descriptive variable names here?
for example `s` in `std::unordered_set <std::string> s`.

Reply:
This section was there originally. the only change that has come from my PR is that adding a vertical space

Reply:
Ok